### ISSUE TITLE
Fix 12 model version/colorscale limit

### ISSUE THREAD

**Issue by yuorme** [yuorme/covid-projections#31]
Currently, the existing colorscales support 12 colors. There are now 13 versions of the IHME model with more updates to come. This is creating problems with plotly's colorscale. Temporary fix is to set the default date range such that 12 IHME models are displayed. This is obviously not a good long-term solution.

I think rather than selecting a date range, we may want to have a slider that allows you to select last N versions of any selected models. 

We can also look towards ways of visualizing more than 12 model versions (maybe using finer grained color gradations or dashed lines) but I think the graph is already busy enough as is.

**Comment by patrickvossler18** [comment]
I like the idea of being able to choose the last N versions. If we want to show all of the versions at once, I think using dashed lines is a better way to go than finer color gradients.

**Comment by yuorme** [comment]
Yeah, I suspect you're probably right. I think there are diminishing returns in showing older models. I've improved the current fix by limiting models to those published within the past 31 days. This should solve the >12 models issue.

So at this point, I'm not really sure if a slider is a better interface than a date range picker. What are your thoughts?
